src/System.Management.Automation/utils/FuzzyMatch.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Based off https://www.csharpstar.com/csharp-string-distance-algorithm/
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Management.Automation
+{
+    internal static class FuzzyMatcher
+    {
+        public const int MinimumDistance = 5;
```

Comment:
Is this settable by the user?

Reply:
No.  It's an implementation detail and if we ever change the algorithm, this setting may not even make sense.